smart-cards/src/main/java/gov/va/api/health/smartcards/R4MetadataController.java

```diff
@@ -0,0 +1,167 @@
+    value = {"/r4/metadata"},
+    produces = {"application/json", "application/fhir+json"})
+@AllArgsConstructor(onConstructor = @__({@Autowired}))
+public class R4MetadataController {
+  private static final String NAME = "API Management Platform | Smart Cards - R4";
+
+  private static final String RESOURCE_DOCUMENTATION =
+      "Implemented per specification. See http://hl7.org/fhir/R4/http.html";
+
+  private final BuildProperties buildProperties;
+
+  private final LinkProperties pageLinks;
+
+  private List<ContactDetail> contact() {
+    return List.of(
+        ContactDetail.builder()
+            .name("API Support")
+            .telecom(
+                List.of(
+                    ContactPoint.builder()
+                        .system(ContactPoint.ContactPointSystem.email)
+                        .value("api@va.gov")
+                        .build()))
+            .build());
+  }
+
+  private CapabilityStatement.Implementation implementation() {
+    return CapabilityStatement.Implementation.builder()
+        .description(NAME)
+        .url(pageLinks.r4Url())
+        .build();
+  }
+
+  @GetMapping
+  CapabilityStatement read() {
+    return CapabilityStatement.builder()
+        .resourceType("CapabilityStatement")
+        .id("smart-cards-capability-statement")
+        .version("1.0.2")
+        .name(NAME)
+        .title(NAME)
+        .publisher("Department of Veterans Affairs")
+        .status(CapabilityStatement.Status.active)
+        .implementation(implementation())
+        .experimental(true)
+        .contact(contact())
+        .date(buildProperties.getTime().toString())
+        .description("Read and search support for credentials of immunization by patient.")
+        .kind(CapabilityStatement.Kind.capability)
+        .software(software())
+        .fhirVersion("4.0.1")
+        .format(List.of("application/json", "application/fhir+json"))
+        .rest(rest())
+        .build();
+  }
+
+  private List<CapabilityStatement.CapabilityResource> resources() {
+    return Stream.of(
+            SupportedResource.builder()
+                .type("Patient")
+                .profileUrl("https://www.hl7.org/fhir/r4/patient.html")
+                .build())
+        .map(SupportedResource::asResource)
+        .collect(toList());
+  }
+
+  private List<CapabilityStatement.Rest> rest() {
+    return List.of(
+        CapabilityStatement.Rest.builder()
+            .mode(CapabilityStatement.RestMode.server)
+            .resource(resources())
+            .build());
+  }
+
+  private CapabilityStatement.Software software() {
+    return CapabilityStatement.Software.builder()
+        .name(buildProperties.getGroup() + ":" + buildProperties.getArtifact())
+        .releaseDate(buildProperties.getTime().toString())
+        .version(buildProperties.getVersion())
+        .build();
+  }
+
+  @Getter
+  @AllArgsConstructor
+  enum SearchParam {
+    _ID("_id", CapabilityStatement.SearchParamType.token);
```

Comment:
Hmm, for this app we don't actually support any searches...

Reply:
🤦 is all I can say.